hack/sync-manifests.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
```

Comment:
Thank you for adding sync script!
@kubeflow/wg-training-leads @Electronic-Waste Appreciate your review as well!